apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -42,7 +39,7 @@ export default class TeacherHomepage extends Component {
 
   render() {
     const { courses, topCourse, announcements, isRtl, queryStringOpen, joinedSections } = this.props;
-    const { canViewAdvancedTools, hocLaunch, isEnglish } = this.props;
```

Comment:
If you don't need `isEnglish` here, do you still need it in PropTypes?

Reply:
I deliberately left it there because it was plumbed through and seemed like it might be useful again in the future, but you're right that we're not using it right now.

Reply:
My preference would be to remove these when they're no longer used.  If we added this back in the future ideally it would be connected via Redux.